src/test/java/org/alfresco/service/cmr/repository/datatype/DefaultTypeConverterTest.java

```diff
@@ -118,8 +120,12 @@ public void testToString()
         // VersionNumber
         assertEquals("1.2.3", DefaultTypeConverter.INSTANCE.convert(String.class, new VersionNumber("1.2.3")));
         // Period
-        assertEquals("period", DefaultTypeConverter.INSTANCE.convert(String.class, new Period("period")));
+        //assertEquals("period", DefaultTypeConverter.INSTANCE.convert(String.class, new Period("period")));
```

Comment:
Has this been intentionally commented out?

Reply:
If the code is no longer required, it should be deleted.